lib/css/components/sidebar-widgets.less

```diff
@@ -0,0 +1,245 @@
+.s-sidebarwidget {
+    --_sw-content-px: calc(var(--su16) - var(--su1)); // subtract 1px for border
+    --_sw-content-spacing-inner: var(--su12); // the spacing between two adjacent simple items
+    --_sw-content-spacing-outer: var(--su16); // the spacing at the start/end of a group of simple items, as well as between a complex item and its separator line
+    --_sw-bc: var(--bc-medium);
+    --_sw-content-bc: var(--bc-light);
+    --_sw-header-bg: var(--black-025);
+
+    & &--header {
```

Comment:
I've been taking the simple approach of alphabetizing each grouping of elements. It's the simplest way to order and allows for easy jumping to what you might be looking for. 

_That said_, it's something I don't think we need to be dogmatic about and I'm open to whatever ordering makes sense in context. @giamir Did you have a specific order for these child elements?

Reply:
no, I just left the same order they were in when I started the refactoring. Will address this and order alphabetically.